dev-latest-kubeflow-org/deploy_latest.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+# Required Variables
+export KUBEFLOW_REPO=${KUBEFLOW_REPO:-}
+export PROJECT=${PROJECT:-}
```

Comment:
We should also parameterize the kubernetes namespace. Otherwise it defaults to "kubeflow".

Reply:
Thanks for pointing out!